pylibs/otns/cli/OTNS.py

```diff
@@ -707,6 +709,17 @@ def get_poll_period(self, nodeid: int) -> float:
         ms = self._expect_int(self.node_cmd(nodeid, 'pollperiod'))
         return ms / 1000.0
 
+    def coaps_enable(self) -> None:
+        self._do_command('coaps enable')
+
+    def coaps(self) -> List[Dict]:
+        """
+        Get recent COAP messages.
```

Comment:
```suggestion
        Get recent CoAPs messages.
```

Reply:
I think `COAP messages` is more right. It has nothing to do with COAPS